lib/config/schema.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Angular CLI Config Schema",
+  "type": "object",
+  "properties": {
+    "project": {
+      "description": "The global configuration of the project.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+      },
+      "additionalProperties": false
+    },
+    "apps": {
+      "description": "Properties of the different applications in this project.",
+      "type": "array",
+      "items": {
```

Comment:
This `items` is missing the `additionalProperties` property. (Not sure if it was intentional, but it seems to be the only one.)